cluster/src/main/scala/org/apache/spark/ui/SnappyDashboardPage.scala

```diff
@@ -744,10 +744,10 @@ object SnappyDashboardPage{
   tableStatsColumn += ("distributionTypeTooltip" -> "Distribution Type is either PARTITIONED or REPLICATED table ")
   tableStatsColumn += ("rowCount" -> "Row Count")
   tableStatsColumn += ("rowCountTooltip" -> "Total Rows in Table")
-  tableStatsColumn += ("sizeInMemory" -> "In Memory Size")
+  tableStatsColumn += ("sizeInMemory" -> "Memory Size")
   tableStatsColumn += ("sizeInMemoryTooltip" -> "Tables Size in Memory")
   tableStatsColumn += ("totalSize" -> "Total Size")
-  tableStatsColumn += ("totalSizeTooltip" -> "Total Size of Tables")
+  tableStatsColumn += ("totalSizeTooltip" -> "Tables Total Size (In Memory size + Disk Overflow Size)")
```

Comment:
Hi Sumedh,

This is a tool tip text and needs to be specific. Similar scheme is used in members tabular list also on UI.